test/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivityDaoTest.java

```diff
@@ -1,456 +0,0 @@
-                endDateTime, history.getNextPageOffsetKey(), SMALL_PAGE_SIZE);
-        assertExpectedUniqueItemsReturned(allTaskGuids, history, SMALL_PAGE_SIZE);
-        
-        // Get fifth page of records. We are definitely paging correctly because GUIDs continue to be unique
-        history = activityDao.getActivityHistoryV3(healthCode, ActivityType.TASK, referentGuid, startDateTime,
-                endDateTime, history.getNextPageOffsetKey(), SMALL_PAGE_SIZE);
-        assertExpectedUniqueItemsReturned(allTaskGuids, history, SMALL_PAGE_SIZE);
-        
-        // Query for a time range that will produce no records
-        history = activityDao.getActivityHistoryV3(healthCode, ActivityType.TASK, referentGuid, startDateTime,
-                startDateTime, null, SMALL_PAGE_SIZE);
-        assertEquals(0, history.getItems().size());
-        assertNull(history.getNextPageOffsetKey());
-    }
-    
-    @Test
-    public void getScheduledActivityHistoryV3NormalPaging() {
-        DateTime startDateTime = DateTime.now(MSK).minusDays(20);
-        DateTime endDateTime = DateTime.now(MSK).plusDays(20);
-        
-        ScheduleContext context = new ScheduleContext.Builder()
-            .withHealthCode(healthCode)
-            .withStudyIdentifier(TEST_STUDY_IDENTIFIER)
-            .withClientInfo(ClientInfo.UNKNOWN_CLIENT)
-            .withInitialTimeZone(MSK)
-            .withStartsOn(startDateTime)
-            .withEndsOn(endDateTime)
-            .withEvents(eventMap()).build();
-        
-        makeSchedulePlanWithSameActivity(context);
-        makeSchedulePlanWithSameActivity(context);
-        
-        String referentGuid = TestUtils.getActivity3().getTask().getIdentifier();
-        
-        Set<String> allTaskGuids = Sets.newHashSet();
-        
-        // Get the first page of records
-        ForwardCursorPagedResourceList<ScheduledActivity> history = activityDao.getActivityHistoryV3(
-                healthCode, ActivityType.TASK, referentGuid, startDateTime, endDateTime, null, MID_PAGE_SIZE);
-        assertExpectedUniqueItemsReturned(allTaskGuids, history, MID_PAGE_SIZE);
-
-        // Get second page of records
-        history = activityDao.getActivityHistoryV3(healthCode, ActivityType.TASK, referentGuid, startDateTime,
-                endDateTime, history.getNextPageOffsetKey(), MID_PAGE_SIZE);
-        assertExpectedUniqueItemsReturned(allTaskGuids, history, MID_PAGE_SIZE);
-        
-        // Get third page of records. These continue to return unique guids.
-        history = activityDao.getActivityHistoryV3(healthCode, ActivityType.TASK, referentGuid, startDateTime,
-                endDateTime, history.getNextPageOffsetKey(), MID_PAGE_SIZE);
-        assertExpectedUniqueItemsReturned(allTaskGuids, history, MID_PAGE_SIZE);
-    }
-
-    @Test
-    public void getScheduledActivityHistoryV3InvalidOffsetKey() {
```

Comment:
Similarly, should this be copied to integ tests?

Reply:
See above